inst/rstudio/templates/project/RcppParallel.package.skeleton.dcf

```diff
@@ -0,0 +1,10 @@
+Binding: RcppParallel.package.skeleton
+Title: R Package using RcppParallel
+Subtitle: Create a new R Package using RcppParallel
+Caption: Create R Package using RcppParallel
+OpenFiles: Read-and-delete-me, src/vector-sum.cpp
```

Comment:
I agree, but in this case we're building off of `package.skeleton()` and `Rcpp.package.skeleton()`, which both generate this file. I figured that, since `RcppParallel` is part of the `Rcpp` family of packages, we ought to stick with those conventions. (Let me know if you think we should diverge from those skeleton functions and provide our own implementation / just clean up that file)